### DISCUSSION TITLE
Querying Features

### DISCUSSION THREAD

**Discussion by wroge** [go-spatial/tegola#864] [General]
I currently use Geoserver for map related things, but I would like to switch to Tegola. Currently Tegola does not provide a way to query features (WFS), so wondering if there is already consideration to provide something like this in the future? Are there other tools you guys use for these things?
Thank you for your great work.

**Comment by ARolek** [comment]
@wroge we don't plans to turn tegola into a WFS server right now. As for other options, we did have a project going years ago but it has since been archived: [jivan](https://github.com/go-spatial/jivan). If you're interested in contributing it would be great to have that project resurface. Another option is [pg_featureserv](https://github.com/CrunchyData/pg_featureserv) by the great folks @CruncyData
